package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -300,7 +300,15 @@ bynum *index-range*
     selects atoms 5 through 10 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 1 up to the total number of atoms.
-
+    
+id *index-range*
+    id is a topology attribute which can be labeled according to the user
+    and conventionally corresponds to serial number in PDB format.
+    selects all atoms in the given range which consists of two numbers 
```

Comment:
Capitalize the first word of a document and the first word after a period.

```suggestion
    Selects all atoms in the given range which consists of two numbers 
```

Reply:
Actually it wasn't capitalized in bynum and index so i left it the same